src/app.ts

```diff
@@ -192,6 +205,11 @@ async function app() {
     clr('║') + ' ├────────────────────┼───────────────────┼────────────────────┤ ' + clr('║'),
     clr('║') + ` │ ${    instNum    } │ ${    invNum    } │ ${    mapNum     } │ ` + clr('║'),
     clr('║') + ' ╰────────────────────┴───────────────────┴────────────────────╯ ' + clr('║'),
+    clr('║') + ' ╭────────────────────┬───────────────────┬────────────────────╮ ' + clr('║'),
+    clr('║') + ' │     Aliases        │                   │                    │ ' + clr('║'),
```

Comment:
Nitpicky, but: `Aliases` is not centered properly in the block.

Reply:
Also, I can't decide for sure if it's actually very helpful to output the number of aliases or not.  I'd be OK w/ just deleting this row too.

Or keep it and center `Aliases` in its box.

Either way.